src/reducers/quiz.js

```diff
@@ -1,8 +1,8 @@
-import quizData from '../data/quiz.json';
+import {quiz} from '../data/index.json';
```

Comment:
can you use `data.json` instead of `data/index.json` ?

Reply:
have you applied prettier here?

Reply:
Yes it is necessary. I will update in the next commit